release-notes/v1_39.md

```diff
@@ -65,6 +65,16 @@ With the introduction of a new variable `${defaultBuildTask}` it is now possible
 
 TODO@isidorn https://github.com/microsoft/vscode/issues/68084
 
+## Contributions to extensions
+
+### Java 13 support 
+
+[Java 13](http://jdk.java.net/13/) is out and VS Code is ready for it. It supports Java 13 through latest [Java Extension](https://marketplace.visualstudio.com/items?itemName=redhat.java). For developers use Java 12 with preview features, you will need to migrate to 13 to keep working with them.
```

Comment:
Do you happen to know some documentation that describes how to migrate to 13? A link here would be very helpful.